### ISSUE TITLE
Unexpected token =

### ISSUE THREAD

**Issue by tacticscale** [zachweinberg/react-native-coinbasepro-clone#1]
git clone <the link>
under the source folder, 
% npx react-native run-ios
npx: installed 643 in 34.009s
Unexpected token =

**Comment by zachweinberg** [comment]
Hi @tacticscale 
This project has not been maintained for a while. Can you please try:

1. Clone fresh project
2. `npm i` inside the project
3. `cd ios/ && pod install`
4. In the root folder, `./node_modules/react-native/cli.js run-ios`

**Comment by tacticscale** [comment]
Yes, and a few others, updated ./ios/Podfile to the following, and then it builds on iPhone 11 simulator:

    # Enables Flipper.
      #
      # Note that if you have use_frameworks! enabled, Flipper will not work and
      # you should disable these next few lines.
      # use_flipper!
      use_flipper!({ 'Flipper-Folly' => '2.5.3', 'Flipper' => '0.87.0', 'Flipper-RSocket' => '1.3.1' })

      post_install do |installer|
        flipper_post_install(installer)

        installer.pods_project.targets.each do |target|
          target.build_configurations.each do |config|
            config.build_settings.delete 'IPHONEOS_DEPLOYMENT_TARGET'
          end
        end
      end
